src/index.html

```diff
@@ -1,13 +1,20 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Antisnake</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Antisnake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Antisnake</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="icon">
+      <div class="icon__box box--1">1</div>
+      <div class="icon__box box--2">2</div>
+      <div class="icon__box box--3">3</div>
+      <div class="icon__box box--4">4</div>
+      <div class="icon__box box--5">5</div>
+      <div class="icon__box box--6">6</div>
```

Comment:
```suggestion
      <div class="icon__box icon__box--1">1</div>
      <div class="icon__box icon__box--2">2</div>
      <div class="icon__box icon__box--3">3</div>
      <div class="icon__box icon__box--4">4</div>
      <div class="icon__box icon__box--5">5</div>
      <div class="icon__box icon__box--6">6</div>
```

В тебе ж це модифікація елемента, хіба ні?